pages/your_problem/data_licence.md

```diff
@@ -5,36 +5,50 @@ contributors: [Siiri Fuchs, Minna Ahokas]
 search: exclude
 ---
 
-## "Concrete Problem 1, Ideally Formulated as a Question"
+## "Why should I assign a licence to my research data?"
  
-### Description
-
-Short explanation of what this problem is.
+### Description 
+Licensing is an important aspect of meeting the R (reusable) principle in FAIR data management. As part of the publication process, you need to decide under which license the data is released. A license is like a greeting to any person wishing to access, cite and use your data telling what they are allowed to do with the data.
 
 ### Considerations
-
-* Optional field, ideally structured in bullet points, detailing things to consider about this problem
+* Remember that a licence is needed also if you use someone else’s data, that is protected by copyright or related rights.
+* If no license is granted, re-use is not permitted without the consent of the copyright holder.
```

Comment:
This is not always true, since not all data is subject to copyright, see e.g. the "Is data always subject to copyright?" paragraph on https://www.openaire.eu/research-data-how-to-license/

```suggestion
* If no license is granted, re-use is generally not permitted without the consent of the copyright holder.
```

Reply:
Yes agreed! Thank for the addition.